src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLStopImpl.java

```diff
@@ -78,7 +79,19 @@
 
   @Override
   public DataFetcher<String> name() {
-    return environment -> getValue(environment, Stop::getName, Station::getName);
+    return environment -> {
+      String lang = new LegacyGraphQLTypes.LegacyGraphQLStopNameArgs(
+              environment.getArguments()).getLegacyGraphQLLang();
+      return getValue(
+              environment,
+              stop -> {
+                return stop.getName().toString(lang == null ? environment.getLocale() : new Locale(lang));
```

Comment:
I think we could extract a helper for this

Reply:
TranslationHelper class has been made.

Reply:
Can you please remove the duplication in this class?